### DISCUSSION TITLE
What is the weapon type for Construction PDAs, Invis watches, disguise kits, and destruction PDAs?

### DISCUSSION THREAD

**Discussion by Gamepro5** [nosoop/SM-TFCustomWeaponsX#46] [Q&A]
I thought that the construction pda and the disguise kit were both "pda" while the destruction PDA and invis watch is "pda2". These values don't seem to work with CWX, so they must be wrong. What are they?
```
"Items"
{
"Cloak"
{
"name" "Cloak"
"inherits" "Upgradeable TF_WEAPON_BUILDER_SPY"
"used_by_classes"
{
"spy" "pda2"
}
"attributes_game"
{
"mult cloak meter consume rate" "5"
}
}
"Building Gaming"
{
"name" "Building Gaming"
"inherits" "Upgradeable TF_WEAPON_BUILDER"
"used_by_classes"
{
"engineer" "pda"
}
"attributes_game"
{
"max health additive bonus" "999"
}
}
}

```

**Comment by nosoop** [comment]
It'll match whatever is specified in the schema (`items_game.txt`):

- Build PDA uses `pda`
- Destroy PDA uses `pda2`
- Sapper uses `building`
- Disguise Kit uses `pda`
- Invis Watch uses `pda2`

Side note:  You don't need to specify `used_by_classes` if the item specified by `inherits` would normally be in the slot for that class.

**Reply by Gamepro5** [reply]
I have those variables set. Should I open an issue it it still doesn't work? I can select the spy watch but cannot use it, and I cannot even select the engineer PDAs.

**Reply by nosoop** [reply]
`Upgradeable TF_WEAPON_BUILDER_SPY` is a sapper, so it should be under `building`.  `TF_WEAPON_INVIS` is what you want if you want a stock Invis Watch replacement.

`Upgradeable TF_WEAPON_BUILDER` is not a valid item &mdash; CWX will have warned you of this in the server console when the schema was read:
```
[cwx/cwx.smx] Item uid 'Building Gaming' inherits from unknown item 'Upgradeable TF_WEAPON_BUILDER'
```

**Reply by Gamepro5** [reply]
Oh! My bad, I'll fix that on the site right away. Thanks for the help!
